AttitudeManager/ControlAlgorithms/Inc/AM_FixedControl.hpp

```diff
@@ -36,6 +38,37 @@ class FixedControl : public ControlInterface {
 
   private:
     const ActuatorConfig configs[NumActuatorIdx];
+    
+    // confirm values before flight ?
+    // this entire secion feels bloated. Improvements ?
```

Comment:
I think it makes sense to load these as runtime configurable values, since the rest of the state mix is also runtime configurable.

I think leaving this as is for now is probably fine though, as long as we capture the work that needs to be done in Asana or something.

Reply:
Could you elaborate on "runtime configurable"? Or point me to a document that explains it well :)

Reply:
this is going to be addressed soon, we just had a meeting about it today